cmd/nerdctl/image_history.go

```diff
@@ -208,31 +218,47 @@ func printHistory(cmd *cobra.Command, historys []historyPrintable) error {
 	return nil
 }
 
-func (x *historyPrinter) printHistory(p historyPrintable) error {
+func (x *historyPrinter) printHistory(printable historyPrintable) error {
```

Comment:
HI @apostasie 

Is it better to add a test case for it :-)

Reply:
But but but but. Who needs tests! Testing is doubting! :D :D :D 


Ok - I'll have a look and get tests in with the next push. Thanks for keeping me honest :-).